### PR TITLE
Test /wallet/transactions

### PR DESCRIPTION
**Pull request by Vavilen**
For [#641](https://github.com/skycoin/skycoin/issues/641)

### CODE REVIEW

File: src/gui/wallet_test.go [hunk 1]
```diff
@@ -555,3 +561,113 @@ func TestWalletGet(t *testing.T) {
...
+			"/wallet/transactions",
+			&httpBody{
+				Id: "foo",
+			},
+			http.StatusBadRequest,
+			"400 Bad Request - get wallet unconfirmed transactions failed: gateway.GetWalletUnconfirmedTxns error",
+			"foo",
+			make([]visor.UnconfirmedTxn, 0),
+			errors.New("gateway.GetWalletUnconfirmedTxns error"),
+			[]visor.UnconfirmedTxn{},
```
Comment (gz-c) [inline src/gui/wallet_test.go]: can you change this to check for `wallet.ErrNotExist` and 404, otherwise 500

**Comment by Vavilen** [comment]
@gz-c , fixed

**Comment by gz-c** [comment]
merge conflicts

File: src/gui/wallet_test.go [hunk 2]
```diff
@@ -845,3 +851,127 @@ func TestUpdateWalletLabelHandler(t *testing.T) {
...
+		url                                   string
+		body                                  *httpBody
+		status                                int
+		err                                   string
+		walletId                              string
+		gatewayGetWalletUnconfirmedTxnsResult []visor.UnconfirmedTxn
+		gatewayGetWalletUnconfirmedTxnsErr    error
+		responseBody                          []visor.UnconfirmedTxn
+	}{
+		{
```
Comment (gz-c) [inline src/gui/wallet_test.go]: `MethodPost`?

**Pull request merged**